### DISCUSSION TITLE
'deepxde.data' has no attribute 'OpDataSet'

### DISCUSSION THREAD

**Discussion by anshumansinha16** [lululxvi/deepxde#867] [Q&A]
I am trying to work with DeepOnet for my project. I am deifining the following code
```
def DeepONet(samples, split, y, I, inds, neurons, epochs, b_layers):
    
    import deepxde as dde
```
```
    X_train = (X_train0, X_train1) # Defined X_train, X_test above this line, but not shown here for brevity.
    X_test = (X_test0, X_test1)
    
    data = dde.data.OpDataSet(
    X_train=X_train, y_train=y_train, X_test=X_test, y_test=y_test )
```

The following error is arising when I'm trying to

> Traceback (most recent call last):
>   File "/Users/anshumansinha/Desktop/ML_project_Spencer/deepOscillations_try-master/./main.py", line 300, in <module>
>     NN_MSEs_test, NN_MSEs_train = DeepONet(samples, split, y/np.max(np.abs(y)) , I, inds, neurons, epochs, b_layers)
>   File "/Users/anshumansinha/Desktop/ML_project_Spencer/deepOscillations_try-master/./main.py", line 246, in DeepONet
>     data = dde.data.OpDataSet( AttributeError: module 'deepxde.data' has no attribute 'OpDataSet'

**Comment by lululxvi** [comment]
See https://deepxde.readthedocs.io/en/latest/demos/operator.html
